packages/mpl-image-labeller.yml

```diff
@@ -0,0 +1,5 @@
+repo: ianhi/mpl-image-labeller
+keywords: image, classification, label, labeler, labeller
+section: interactivity
+description: Easily (label) annotate images with for training classification models.
```

Comment:
- can you make the sentence grammatical?  Also, this makes it sound like you are annotating the image, which is not _really_ what you are doing, is it?